server/src/queues/everyHourQueue.ts

```diff
@@ -0,0 +1,21 @@
+import Queue from "bull";
+require("dotenv").config();
+import { logger } from "../logger";
+import everyHourQueueProcessors from "../processors/everyHourQueueProcess";
+
+const everyHourQueue = new Queue("everyHourQueue", "redis://127.0.0.1:6379");
+
+everyHourQueue.process(everyHourQueueProcessors);
+
+const addQueueEveryHour = () => {
+  everyHourQueue
+    .add(
+      {},
+      {
+        repeat: { cron: `0 * * * *` },
+      }
+    )
+    .then(() => logger.debug("Que to run every hour added"));
```

Comment:
```suggestion
    .then(() => logger.debug("Queue to run every hour added"));
```
Also add catch block